qoqo_for_braket/qoqo_for_braket/backend.py

```diff
@@ -523,7 +599,7 @@ def _run_measurement_registers_hybrid(
         shutil.copyfile(
             helper_file_path, os.path.join("_tmp_hybrid_helper", "qoqo_hybrid_helper.py")
         )
-        requirement_lines = ["qoqo >= 1.9\n", "qoqo-for-braket >= 0.4"]
+        requirement_lines = ["qoqo >= 1.11\n", "qoqo-for-braket >= 0.5"]
```

Comment:
Just a reminder that if we decide to update to qoqo 1.4 we'll need to change it here as well

Reply:
Same as above.